crates/bevy_core_pipeline/src/main_pass_3d.rs

```diff
@@ -134,12 +134,18 @@ impl Node for MainPass3dNode {
                 })],
                 depth_stencil_attachment: Some(RenderPassDepthStencilAttachment {
                     view: &depth.view,
-                    // NOTE: For the transparent pass we load the depth buffer but do not write to it.
-                    // As the opaque and alpha mask passes run first, opaque meshes can occlude
-                    // transparent ones.
```

Comment:
Please keep this note but update it to be correct.

Reply:
I added back the comment with modifications to indicate that the reasoning about the lack of write-back is now non-local.

Reply:
Nice! :)